src/contract_wrappers/modules/wallet/vesting_escrow_wallet_wrapper/index.ts

```diff
@@ -1,12 +1,40 @@
 /* istanbul ignore file */
+import {
+  VestingEscrowWalletEvents_3_1_0,
+  VestingEscrowWalletEvents_3_0_0,
+  VestingEscrowWalletEventArgs_3_0_0,
+} from '@polymathnetwork/abi-wrappers';
 import { VestingEscrowWallet_3_0_0, isVestingEscrowWallet_3_0_0 } from './3.0.0';
 import { VestingEscrowWallet_3_1_0, isVestingEscrowWallet_3_1_0 } from './3.1.0';
 
 import Common, { isVestingEscrowWallet } from './common';
-import { ContractVersion, Subscribe, GetLogs } from '../../../../types';
+import { ContractVersion } from '../../../../types';
+
+export const VestingEscrowWalletEvents = {
```

Comment:
Why is this necessary if VEW doesn't have changes to events in 3.1?

Reply:
It isn't necessary but I applied it as a good practice. It doesn't harm neither because it only takes 3.0 events, but it we do this always for each new release, it will prevent us to forget some new event.